backend/drive/drive.go

```diff
@@ -542,6 +543,8 @@ type Fs struct {
 	exportExtensions []string           // preferred extensions to download docs
 	importMimeTypes  []string           // MIME types to convert to docs
 	isTeamDrive      bool               // true if this is a team drive
+
+	configMapper configmap.Mapper // TODO: Figure a way around this
```

Comment:
My preference would be to call this just `m` like the rest of the code. Terse I know but conventions count!

I don't have a problem with you storing it in the config - seems like a reasonable thing to do (the oauthutil stuff does so that it can write back to the config file).